### PR TITLE
Newsletters: add 292 (2024-03-06)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries @harding

### CODE REVIEW

File: _posts/en/newsletters/2024-03-06-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  used to transmit privacy-respecting persistent identifiers, such as
+  those for [silent payments][topic silent payments] and [offers][topic
+  offers].
+
+    As discussed on the thread, an improvement could be having the
+    creator of the URI specify all of the payment methods they supported
+    using bare parameters, for example: `bitcoin:?bc1q...&sp1q...`.  The
+    spender (who is usually responsible for paying fees) could then
+    choose their preferred payment method from the list.  Although some
+    minor technical points were still being discussed at the time
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-03-06-newsletter.md:36]:
```suggestion
    minor technical points were still being discussed at the time of
```

File: _posts/en/newsletters/2024-03-06-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  to add to a transaction they create.  With many hardware signing
+  devices limited in the amount of storage they have available, it would
+  be very useful to minimize the amount of state required (without
+  reducing security).
+
+  Ingala proposes generating a single state object for an entire
+  [PSBT][topic psbt] and then deterministically deriving the per-input
+  state from it in a way that still makes the result indistinguishable
+  from random.  That way the amount of state a signer needs to store is
+  constant no matter how inputs a transaction has.
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-03-06-newsletter.md:54]:
```suggestion
  constant no matter how many inputs a transaction has.
```

File: _posts/en/newsletters/2024-03-06-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  clear resolution had been reached as of this writing.
+
+- **GitLab backup for Bitcoin Core GitHub project:** Fabian Jahr
+  [posted][jahr gitlab] to Delving Bitcoin about maintaining a backup of
+  the Bitcoin Core project's GitHub account on an self-hosted GitLab
+  instance.  In case the project ever needed to leave GitHub suddenly,
+  this would make all existing issues and pull requests accessible on
+  GitLab within a short amount of time, allowing work to continue with
+  only a brief interruption.  Jahr provided a preview of the project on
+  GitLab and plans to keep backups going forward to allow rapidly
```
[the remark this answers is not in the document]
Comment (vostrnad) [reply _posts/en/newsletters/2024-03-06-newsletter.md:75]: Seems fine as is.

File: _posts/en/newsletters/2024-03-06-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+- [Bitcoin Core #29412][] adds code that checks for every known way to
+  take a valid block, mutate it, and produce an alternative block that
+  is invalid but has the same block header hash.  Mutated blocks have
+  caused multiple vulnerabilities in the past.  In 2012, and again in
+  2017, Bitcoin Core's cached rejection of invalid blocks meant that an
+  attacker could take a new valid block, mutate it into an invalid
+  block, and submit it to a victim's node; that node would reject it as
+  invalid and would not later accept the valid form of the block (until
+  the node was next restarted), forking it off the best block chain and
```
[the remark this answers is not in the document]
Comment (harding) [reply _posts/en/newsletters/2024-03-06-newsletter.md:109]: Leaving as-is for this newsletter.  I left a comment on #1541 and, if there's no major disagreement there, I'll switch to "blockchain" going forward.

**Comment by harding** [comment]
ACK da85a2d67d4fdf7f3f606596958ec7fa1b941a9a

If there are no objections, I plan to merge this on Sunday, March 10th, before opening the PR for next week's newsletter.

**Comment by harding** [comment]
Made edits for all feedback, except as replied, and added remaining content.  Thanks everyone!

File: _posts/en/newsletters/2024-03-06-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Eclair v0.10.0][] is a new major release of this LN node
+  implementation.  It "adds official support for the [dual-funding
+  feature][topic dual funding], an up-to-date implementation of Bolt 12
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-03-06-newsletter.md:96]:
```suggestion
  feature][topic dual funding], an up-to-date implementation of BOLT12
```

**Comment by bitschmidty** [comment]
Squashed and merged, thank you @harding for authoring and @LarryRuane and @vostrnad for the review this week 🚀

**Pull request merged**